pack/return/rttcu.json

```diff
@@ -1,52 +1,254 @@
+        "traits": "Tarot.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "54002",
+        "cost": 2,
+        "deck_limit": 1,
+        "faction_code": "guardian",
+        "illustrator": "John Pacer",
+        "name": "Hallowed Mirror",
+        "pack_code": "rttcu",
+        "position": 2,
+        "quantity": 1,
+        "slot": "Accessory",
+        "skill_willpower": 2,
+        "text": "Limit 1 per deck.\n[reaction] When you play a copy of Soothing Melody: Either change each \"2\" on it to a \"3,\" or shuffle it into your deck instead of discarding it.\n</b> - After Hallowed Mirror enter play: Search your bonded cards for 3 copies of Soothing Melody. Add 1 to your hand and shuffle the other 2 into your deck.",
+        "traits": "Item. Relic. Occult. Blessed.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "54003",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "illustrator": "Robert Laskey",
+        "name": "The World • XXI",
+        "pack_code": "rttcu",
+        "position": 3,
+        "quantity": 2,
+        "slot": "Tarot",
+        "subname": "The Journey is Complete",
+        "text": "[fast] If you have 8 or more cards in your hand, exhaust The World • XXI: Draw 1 card.\n[reaction] When the game begins, if The World • XXI is in your opening hand: Put it into play.",
+        "traits": "Tarot.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "54004",
+        "cost": 2,
+        "deck_limit": 1,
+        "faction_code": "seeker",
+        "illustrator": "Dimitri Bielak",
+        "name": "Occult Lexicon",
+        "pack_code": "rttcu",
+        "position": 4,
+        "quantity": 1,
+        "skill_intellect": 2,
+        "slot": "Hand",
+        "text": "Limit 1 per deck.\n[reaction] When you play a copy of Blood-Rite: Either change each \"2\" on it to a \"3,\" or shuffle it into your deck instead of discarding it.\n<b>Forced</b> - After Occult Lexicon enters play: Search your bonded cards for 3 copies of Blood-Rite. Add 1 to your hand and shuffle the other 2 into your deck.",
+        "traits": "Item. Tome. Occult.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "54005",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "rogue",
+        "illustrator": "Robert Laskey",
+        "name": "Knight of Swords",
+        "pack_code": "rttcu",
+        "position": 5,
+        "quantity": 2,
+        "slot": "Tarot",
+        "subname": "Charge Ever Onward",
+        "text": "[reactionn] When you would succeed at a skill test: You get +1 skill value for this test. You may discard Knight of Swords to get +3 instead.\n[reaction] When the game begins, if Knight of Swords is in your opening hand: Put it into play.",
```

Comment:
Typo: reaction has two Ns at the end.

Reply:
thanks, my keyboard is breaking.  #apple problems.